src/Components/OperationsGroup.jsx

```diff
@@ -7,7 +7,7 @@ import ShortcutsControl from './ShortcutsControl'
 import {TooltipIconButton} from './Buttons'
 import CutPlaneIcon from '../assets/2D_Icons/CutPlane.svg'
 import ClearIcon from '../assets/2D_Icons/Clear.svg'
-import {useWindowDimensions} from './Hooks'
+import {useIsMobile} from './Hooks'
 
 
```

Comment:
for the operations group this is how it looks like on desktop:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/3433606/158709440-9dec651e-6b16-45df-b334-b9255e1d5ee5.png">
seems the group is top aligned
with the elements selected it looks like this:
<img width="759" alt="image" src="https://user-images.githubusercontent.com/3433606/158709521-228689eb-c8b8-4db6-804f-395c22c8544b.png">
on the mobile all is fine:
<img width="330" alt="image" src="https://user-images.githubusercontent.com/3433606/158709587-e260878e-48fb-444d-a0c8-d84749797658.png">